### ISSUE TITLE
Discussion about default recipes

### ISSUE THREAD

**Issue by samsmithnz** [Maelstromeous/satisfactory-factories#28]
I find myself clicking a lot to select all of the recipes for production - and I wonder if - by default - we should be selecting the default recipe, and changing to alt recipes - which at least in my games, is typically an exception. This would streamline the setting up of new factories in the tool. What do others think?

**Comment by Maelstromeous** [comment]
Yeah that would be pretty nice behaviour. Only issue is with that I can’t simply say “any non alt recipe, make it default” as there’s say for example Residual Fuel. I could get the vast majority by saying “If part name matches recipe description” but there’s certain cases like where Iron Ingots are listed as IngotIron in the game data. 

I like the idea though, it’s just a bit fiddly and has many edge cases, but it would be a nice time saver.

**Comment by samsmithnz** [comment]
fair comments about the oil/refinery where there is no clear default. Perhaps that is where we add logic to not fill in the defaults there. I'm trying to think if there are other places where this is true.

**Comment by Maelstromeous** [comment]
We can supply intended defaults as a parser step if needed, we could employ a "If it doesn't have 'alt' in the name, it is a default but only have 1' and if we need to manually fix the default mapping that works as well.

**Comment by Maelstromeous** [comment]
Lots of people asking for this on Reddit, P2.
